go/billing/settings.py

```diff
@@ -42,9 +42,9 @@
 
 STATEMENT_CONTACT_DETAILS = getattr(
     settings, 'BILLING_STATEMENT_CONTACT_DETAILS', {
-        'tel': '27.11.482.8684',
-        'email': 'accounts@praekelt.com',
-        'website': 'www.praekeltfoundation.org',
+        'tel': '27.11.123.4567',
+        'email': 'foo@bar.com',
+        'website': 'www.foo.org',
```

Comment:
The officially mandated example domain name is ``example.com`` but ``bar.com`` has quite a fun web page and seems to be a de facto standard. ``www.foo.org`` returns a holding page so we probably shouldn't use it. Thoughts?